src/explore-education-statistics-frontend/server.js

```diff
@@ -112,6 +113,19 @@ async function startServer(port = process.env.PORT || 3000) {
     );
   }
 
+  const cacheTtl = process.env.APP_ENV === 'Local' ? 2 : 10;
+
+  server.get(
+    '/find-statistics',
+    routeCache.cacheSeconds(cacheTtl),
+    (req, res) => handleRequest(req, res),
+  );
+  server.get(
```

Comment:
We'll need the plain /find-statistics/:publication path too, to catch views to the latest Release for a Publication.

Reply:
This route already handles that, it's using an optional `release` path segment.